.github/workflows/integration-tests.yaml

```
@@ -0,0 +1,47 @@
+name: integration-tests
+
+env:
+  AWS_REGION: us-east-1
+  AWS_ROLE_DURATION: 3600
+  # role generated from https://github.com/Sceptre/sceptre-aws/blob/master/config/prod/gh-oidc-sceptre-tests.yaml
+  AWS_ROLE: arn:aws:iam::743644221192:role/gh-oidc-sceptre-tests
+
+on:
+  workflow_run:
+    workflows:
+      - tests
+    branches: "*"
+    types:
+      - completed
+
+jobs:
+  integration-tests:
+    needs:
+      - unit-tests
```

Will this dependency work between separate workflow files? Since this workflow is triggered only by the completed `tests` workflow run, it would only need a single conditional:
```suggestion
    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

---

as i understand it the result of `github.event.workflow_run.conclusion` is from it's dependency defined by `needs` param.